spec/gruf/cli/executor_spec.rb

```diff
@@ -94,15 +94,47 @@
       end
     end
 
+    context 'when services are set via the CLI argument' do
+      let(:options_services) { [] }
+      let(:global_services) { [] }
+      let(:cli_services) { [Rpc::ThingService::Service] }
+      let(:args) { %w[--services Rpc::ThingService::Service] }
+
+      it 'adds each specified service to the server' do
```

Comment:
It would be great if we could populate `options_services` and `global_services` with unique, valid values and ensure that they aren't loaded.